docs/source/intervention_models/crm_mgmt/vivarium_outreach.rst

```diff
@@ -0,0 +1,124 @@
+Vivarium Modeling Strategy - Outreach Intervention 
+--------------------------------------------------
+
+.. todo::
+
+  Add an overview of the Vivarium modeling section.
+
+Decision tree and algorithm for outreach intervention
+
+Visit type: emergency
```

Comment:
I don't think of these visit types as part of the outreach intervention;  I recommend you have a `healtcare_access.rst` to present this information, and then reference the healthcare_access.rst in this doc as relevant.

Reply:
Added visit.rst; linked in the vivarium_{intervention} sections